front/src/app/app.styled.ts

```diff
@@ -1,14 +1,76 @@
 import styled, { createGlobalStyle } from "styled-components";
 
-export const ContainerCustom = styled.div`
+// 
+
+export const CardsContainer = styled.section`
+  display: flex;
+  justify-content: start;
+  align-items: start;
+  gap: 5%;
+  flex-wrap: wrap;
+
+  padding: 1em 5%;
   width: 100vw;
-  height: 10vh;
-`;
+`
+export const CardElement = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content:space-between;
+  margin-bottom: 2em;
+
+  padding: .8em; 
+  height: 20vh;
+  width: 14vw;
+  background-color: #8A9E9F; 
+  border-radius: .4em;
+
+  & .card-image-container{
+    width: 50px;
+    height: 50px;
+    background: #fff;
+    border: solid 2px #434B4D;
+  }
+
+  & .card-upper-section{
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+  }
```

Comment:
esse tipo de coisa vc nao precisa fazer usando o styled compoennts. Vc pode criar um component `CardImageContaner` que seria um div e chamar ele dentro do componente React

```suggestion
const CardImageContainer = styled.div`
  width: 50px;
  height: 50px;
  background: #fff;
  border: solid 2px #434B4D;
`;

const CardUpperSection = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: end;
`;
```